x-pack/apm-server/sampling/processor_test.go

```diff
@@ -590,21 +590,28 @@ func TestGroupsMonitoring(t *testing.T) {
 	})
 }
 
-func getGauge(t testing.TB, reader sdkmetric.Reader, name string) int64 {
+// getGaugeValues collects metrics and searches for gauge values that match the provided names.
+// Values will be returned in the same order as the names.
+//
+// It is helpful to provide multiple names for synchronous metrics to avoid losing data when collecting.
+// Observable metrics report everytime Collect is called, so there will be no data loss.
+func getGaugeValues(t testing.TB, reader sdkmetric.Reader, names []string) []int64 {
```

Comment:
nit, non-blocker: seems to me that it is a good candidate for a variadic function, as name is the last param accepting a slice.

Reply:
Updated thanks